commands/ci/rerun.js

```diff
@@ -32,6 +32,7 @@ function* run (context, heroku) {
       commit_message: sourceTestRun.commit_message,
       commit_sha: sourceTestRun.commit_sha,
       pipeline: pipeline.id,
+      organization: sourceTestRun.organization.name,
```

Comment:
Question: should we reuse the organization specified in the original test run or should we use the org currently set in the pipeline repository?

I can see a case where the initial run is created, then the billing org is changed and then the test is rerun.